vcd/resource_vcd_org_vdc.go

```diff
@@ -263,17 +297,80 @@ func resourceVcdVdcRead(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf(errorRetrievingOrg, err)
 	}
 
-	vdc, err := adminOrg.GetVdcByName(d.Id())
-	if err != nil || vdc == (govcd.Vdc{}) {
-		log.Printf("[DEBUG] Unable to find vdc. Removing from tfstate")
-		d.SetId("")
-		return nil
+	adminVdc, err := adminOrg.GetAdminVdcByName(d.Get("name").(string))
+	if err != nil || adminVdc == (govcd.AdminVdc{}) {
+		log.Printf("[DEBUG] Unable to find vdc")
+		return fmt.Errorf("unable to find vDC %#v", err)
+	}
+
```

Comment:
Some comments about the meaning of these additions would be helpful

Reply:
These change for for full Read - saves latest state from vCD to terraform state file

Reply:
I understand that. What I was asking is to have such comment in the code, as it will be useful for future implementation of similar behavior.

Reply:
Could you elaborate which part isn't self explanatory and needs additional comment. Please.